pkg/test/framework/components/istio/flags.go

```diff
@@ -21,7 +21,19 @@ import (
 // init registers the command-line flags that we can exposed for "go test".
 func init() {
 	flag.StringVar(&settingsFromCommandline.SystemNamespace, "istio.test.kube.systemNamespace", settingsFromCommandline.SystemNamespace,
-		"The namespace where the Istio components reside in a typical deployment.")
+		"Deprecated, specifies the namespace where the Istio components (<=1.1) reside in a typical deployment.")
+	flag.StringVar(&settingsFromCommandline.IstioNamespace, "istio.test.kube.istioNamespace", settingsFromCommandline.IstioNamespace,
+		"Specifies the namespace in which istio ca and cert provisioning components are deployed.")
+	flag.StringVar(&settingsFromCommandline.ConfigNamespace, "istio.test.kube.configNamespace", settingsFromCommandline.ConfigNamespace,
```

Comment:
nit: Control namespace seems more aligned with what we use in the new installer- not sure if that will be user facing or not though

Reply:
@morvencao i'll merge this for now, please addr this comment.  It is not a user facing param but consistency is always good.